modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -4436,4 +4460,78 @@ WifiNetwork = baseclass.extend(/** @lends LuCI.network.WifiNetwork.prototype */
 	}
 });
 
+/**
+ * @class
+ * @memberof LuCI.network
+ * @hideconstructor
+ * @classdesc
+ *
+ * A `Network.WifiVlan` class instance represents a vlan on a WifiNetwork.
+ */
+WifiVlan = baseclass.extend(/** @lends LuCI.network.WifiVlan.prototype */ {
```

Comment:
Perhaps it can just be `Vlan`?

Reply:
I think that might be mistaken for an ethernet vlan interface, which is quite different.

Reply:
Ofc. You're right. It's in `wireless`.